pkg/reconciler/common/read_releases_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	mf "github.com/manifestival/manifestival"
+	util "knative.dev/operator/pkg/reconciler/common/testing"
+)
+
+func TestRetrieveManifestPath(t *testing.T) {
+	koPath := "../../../cmd/operator/kodata"
+
+	tests := []struct {
+		component string
+		version   string
+		label     string
+	}{{
+		component: "knative-serving",
+		version:   "0.14.0",
+	}, {
+		component: "knative-eventing",
+		version:   "0.14.2",
+	}}
+
+	os.Setenv(KoEnvKey, koPath)
+	for _, test := range tests {
+		t.Run(test.component, func(t *testing.T) {
+			manifestPath := RetrieveManifestPath(test.version, test.component)
+			expected := fmt.Sprintf("%s/%s/%s", koPath, test.component, test.version)
```

Comment:
This doesn't make a lot of sense to have here. The `expected` value should go into the test struct above.

Reply:
done